test/runtests.jl

```diff
@@ -28,18 +28,23 @@ MOIU.@model(TestModel,
             (MOI.VectorOfVariables,),
             (MOI.VectorAffineFunction, MOI.VectorQuadraticFunction))
 
-            
+# Functions that are used inside tests
+include("utils.jl")
+
 # Problems database
 include("Problems/Linear/linear_problems.jl")
 include("Problems/Quadratic/quadratic_problems.jl")
 include("Problems/SOC/soc_problems.jl")
+include("Problems/RotatedSOC/rsoc_problems.jl")
```

Comment:
to identify rotated second order cone related code, sometimes you use "rsoc" and sometimes "RotatedSOC". I'm tempted to just suggest using RSOC/rsoc, since the "rotated" doesn't add much useful information to an acronym

Reply:
@blegat @joaquimg I'm probably just being anal though

Reply:
I think it is a good idea!